Formula/potrace.rb

```diff
@@ -3,7 +3,7 @@ class Potrace < Formula
   homepage "https://potrace.sourceforge.io/"
   url "https://potrace.sourceforge.io/download/1.16/potrace-1.16.tar.gz"
   sha256 "be8248a17dedd6ccbaab2fcc45835bb0502d062e40fbded3bc56028ce5eb7acc"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
Is there a specific part of `potrace` that is restricted to `-only`?

At least on homepage (http://potrace.sourceforge.net/#license) & README file

> This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2, or (at your option) any later version.

Reply:
No, I just did not see that particular clause in any of the source files. But seeing as its in the `README`, that looks pretty definitive. Adjusting.